src/routes/Greeting.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreeting } from '../redux/greeting/greetingSlice';
+
+const Greeting = () => {
+  const greeting = useSelector((store) => store.greeting);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchGreeting());
+  }, [dispatch]);
+
+  if (greeting.isLoading) {
+    return <p>Greeting Loading...</p>;
+  }
+
+  return (
+    <div>
+      <h1>Randomized Greetings</h1>
+      <h1>{greeting.greeting.message}</h1>
+    </div>
+  );
```

Comment:
- [ ] Great job working on the frontend side but I think it would be great if you render the messages in the right way since after running the two apps the backend is working fine but the frontend doesn't render the messages it just show me this page so kindly fix the issue to let the messages render as expected 🙌💥♻️


<img width="731" alt="image" src="https://github.com/rmiyoyo/hello-react-front-end/assets/58553711/9c913ca3-8955-4140-ac9c-79290ffb8271">